$NetBSD: patch-ab,v 1.14 2012/08/16 14:19:35 wiz Exp $

Make this package build on Interix.  Patches provided by Hiramatsu
Yoshifumi in PR pkg/25946.
Accepted by Peter Stephenson <p.w.stephenson@ntlworld.com> on
zsh-workers on 2012-08-16.

--- Src/zsh.h.orig	2011-05-16 15:49:11.000000000 +0000
+++ Src/zsh.h
@@ -2122,6 +2122,7 @@ struct ttyinfo {
 #endif
 };
 
+#ifndef __INTERIX
 /* defines for whether tabs expand to spaces */
 #if defined(HAVE_TERMIOS_H) || defined(HAVE_TERMIO_H)
 #define SGTTYFLAG       shttyinfo.tio.c_oflag
@@ -2139,6 +2140,7 @@ struct ttyinfo {
 #   endif
 #  endif
 # endif
+#endif
 
 /* flags for termflags */
 
